wheel/dist_info.py

```diff
@@ -0,0 +1,26 @@
+"""
+Create a wheel (.whl) distribution.
+
+A wheel is a built archive format.
+"""
+
+from setuptools.command.egg_info import egg_info
+
+
+class dist_info(egg_info):
```

Comment:
One problem that I ran into is that even though I inherit from egg_info, I still cannot pass egg_base_option. Do you know why that might be?